lib/rage/middleware/reloader.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class Rage::Reloader
+  def initialize(app)
+    @app = app
+  end
+
+  def call(env)
+    Rage.code_loader.reload
+    @app.call(env)
```

Comment:
Let's also catch errors here.

```suggestion
    @app.call(env)
rescue Exception => e
    exception_str = "#{e.class} (#{e.message}):\n#{e.backtrace.join("\n")}"
    puts(exception_str)
    [500, {}, [exception_str]]
```

Reply:
This was so obvious to catch all exceptions in the middleware itself 🙈 Shame I didn't see this solution but it is really neat!